pilz_robot_programming/src/pilz_robot_programming/robot.py

```diff
@@ -187,8 +188,8 @@ def get_current_joint_states(self, planning_group=_DEFAULT_PLANNING_GROUP):
         try:
             return self._robot_commander.get_group(planning_group).get_current_joint_values()
         except MoveItCommanderException as e:
-            rospy.logerr(e.message)
-            raise RobotCurrentStateError(e.message)
+            rospy.logerr(str(e))
+            raise RobotCurrentStateError(str(e))
```

Comment:
In line 209 its written as "%s" % e and RobotCurrentStateError(e)
We choose one alternative